st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -107,7 +107,13 @@ def _tokenize_alias_execution(self, alias_execution):
         return (tokens[0], tokens[1] if len(tokens) > 1 else None)
 
     def _extract_parameters(self, action_alias_db, format_str, param_stream):
-        if action_alias_db.formats and format_str in action_alias_db.formats:
+        formats = []
+        for formatstring in action_alias_db.formats:
+            if 'representation' in formatstring:
```

Comment:
Good point. Format string can actually be a string with "representation" in it. :) Will fix.